modules/bash-commons/install.sh

```diff
@@ -0,0 +1,82 @@
+#!/bin/bash
+# This script is used by the Gruntwork Installer to install the bash-commons library.
+
+set -e
+
+readonly SCRIPT_DIR="$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)"
+readonly BASH_COMMONS_SRC_DIR="$SCRIPT_DIR/src"
+
+source "$BASH_COMMONS_SRC_DIR/log.sh"
+source "$BASH_COMMONS_SRC_DIR/assert.sh"
+source "$BASH_COMMONS_SRC_DIR/os.sh"
```

Comment:
Ha, you're able to use the bash-commons libs in the bash-commons install script itself!

Reply:
Hehe, yea :)